specs/schema.yaml

```diff
@@ -123,7 +123,7 @@ components:
         maxMsgAge:
           type: "number"
           description: "Maximum age of any message in the channel, expressed in nanoseconds."
-          example: 86400000000
+          example: 8640000000000000
```

Comment:
@yartman saw your PR also updates this, I guess safe would be to merge this PR and then yours.

Reply:
btw: this was updated after running schema:generate

Reply:
sorry, i didn't see this before merge

Reply:
No problem, solved as it triggered merge conflict.